pkg/util/util.go

```diff
@@ -131,6 +131,7 @@ func (u ChartUtils) LookupChartDir(chartDirs []string, dir string) (string, erro
 		for {
 			chartYaml := path.Join(currentDir, "Chart.yaml")
 			parent := path.Dir(path.Dir(chartYaml))
+			chartDir = path.Join(chartDir) // removes any trailing slash from the dir
```

Comment:
if you try setting `chart-dirs` as `./`, the tool gets stuck in an infinite loop. _(even more confusing because `.` works as expected)_ 
This was because of later in the code, 
`if (joined == chartDir) || strings.HasPrefix(relativeDir, "..") {` 
would check if `.` == `./`. 

I didn't test, but i'm pretty sure it was a similar problem with the `(parent == chartDir)` check as well.